src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -162,7 +162,7 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
   const translate = useTranslate()
   const dispatch = useAppDispatch()
   const queryClient = useQueryClient()
-  const { wallet, deviceId: walletDeviceId } = useWallet().state
+  const { wallet, deviceId: walletDeviceId, isDemoWallet } = useWallet().state
```

Comment:
Unrelated to the consolidation of Accounts Management menu, revert me, and possibly bring me back in a follow-up? We shouldn't get into this state in the first place but we may, i.e this guy wrongly opens the account management modal instead of the wallet modal
<img width="1369" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c3cf5e94-efca-470f-a521-ad5daf9a760f">